core/src/main/java/org/elasticsearch/search/aggregations/bucket/terms/support/IncludeExclude.java

```diff
@@ -138,6 +169,29 @@ public boolean accept(BytesRef value) {
 
     }
 
+    class PartitionedOrdinalsFilter extends OrdinalsFilter {
+
+
+        @Override
+        public LongBitSet acceptedGlobalOrdinals(RandomAccessOrds globalOrdinals) throws IOException {
+            final long numOrds=globalOrdinals.getValueCount();
+            final LongBitSet acceptedGlobalOrdinals = new LongBitSet(numOrds);
+            final TermsEnum termEnum = globalOrdinals.termsEnum();
+            
+            BytesRef term = termEnum.next();
+            int ord = 0;
```

Comment:
you need a long, or this could overflow. Or even not store it at all, and use `termsEnum.ord()` to know the ord of the current term